ENet-SAD-Simple/README.md

```diff
@@ -0,0 +1,78 @@
+
+## Requirements
+- [Torch](http://torch.ch/docs/getting-started.html), please follow the installation instructions at [fb.resnet.torch](https://github.com/facebook/fb.resnet.torch).
+
+## Before Start
+
+Please follow [train_final.txt](./list/train_final.txt) and [val_final.txt](./list/val_final.txt) to put LLAMAS dataset in the desired folder. We'll call the directory that you cloned ENet-SAD-Simple as `$ENet_ROOT`.
+
+## Testing
+1. Put your trained model to `./experiments/pretrained`
+    ```Shell
+    cd $ENet_ROOT/experiments/pretrained
+    ```
+   You can just train the model by yourself and have a test.
+
+2. Run test script
+    ```Shell
+    cd $ENet_ROOT
+    sh ./experiments/test.sh
+    ```
+    Testing results (probability map of lane markings) are saved in `experiments/predicts/` by default.
+
+3. Submit the results to server 
+    Please follow the instructions of LLAMAS to submit your results.
+    The performance of our trained model is as follows:
+
+  0: {'auc': 0.9999917047921401,
```

Comment:
https://unsupervised-llamas.com/llamas/benchmark_multi
Feel free to also link to the official benchmark. The results will be up before this PR is merged